### ISSUE TITLE
SVG Icon and Splash Problems

### ISSUE THREAD

**Issue by sretz** [dotnet/maui#9322]
### Description

I had someone create an SVG for my program which I'm about to publish, although the SVG works correctly in MS Edge, Inkview and Inkscape the build process creates problems with the SVG.

Splash Screen in android emulator
![image](https://user-images.githubusercontent.com/80936576/183936446-4fd986da-de04-4d7d-b341-e72e0ee5780a.png)
 
SVG in MS Edge
![image](https://user-images.githubusercontent.com/80936576/183937096-5242e37c-17a4-428c-9c07-054b4bff91c2.png)

App Icon in android emulator
![image](https://user-images.githubusercontent.com/80936576/183937949-51e77986-505d-4038-8355-1f3ca48a5ef8.png)

Another problem is that on my physical Android phone, Samsung SM-G996B (Android 12.0 - API 31), the Splash Screen is using a portion of an icon image.
Splash Screen from Samsung SM-G996B
![image](https://user-images.githubusercontent.com/80936576/183938931-d5c25263-3e1f-45e3-a3b2-54e5f54ef41e.png)

### Steps to Reproduce

1. Download https://github.com/sretz/Reproduce.SVGIconSplashProblems
2. Run in Android emulator
3. Compare SVG in splash screen to SVG in MS Edge

### Version with bug

6.0.486 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Andriod Pixel 5 - API 30 (Andriod 11), Andriod SM-G996B (Andriod 12 - API 31))

### Did you find any workaround?

No

**Comment by sretz** [comment]
I got the svg converted to a png and still have the same problem.

<Project Sdk="Microsoft.NET.Sdk">
	<ItemGroup>
		
		<MauiIcon Include="Resources\appicon.png" ForegroundFile="Resources\appiconfg.png" Color="#808080" />

		
		<MauiSplashScreen Include="Resources\appiconfg.png" Color="#808080" BaseSize="384,384" />
	</ItemGroup>
</Project>

Splash Screen from png
![image](https://user-images.githubusercontent.com/80936576/184389140-fabc9c19-a12e-4038-a998-5e494783215d.png)

PNG from picture viewer
![image](https://user-images.githubusercontent.com/80936576/184389462-29d0d332-cf55-439a-a1de-0a3a00d7113f.png)

appicon.png
![appicon](https://user-images.githubusercontent.com/80936576/184389750-c8eb280d-6efe-450d-99eb-57c8dc5f4299.png)

appiconfg.png
![appiconfg](https://user-images.githubusercontent.com/80936576/184389944-62db6fb9-287f-4f88-9580-526e7b6cc634.png)

**Comment by sretz** [comment]
found a work around of using jpg

**Comment by sretz** [comment]
Although the splash screen is working correctly on my android emulator, I'm still only getting a small portion of the icon on my physical Samsung Android phone as before, but the picture doesn't have glitches in it as before.

**Comment by xiz0r** [comment]
I have the same problem but in the simulator, in the physical device (xiaomi) the splash works fine, that's why I lowered the priority to this... 
I don't have a samsung device to test it....

![image](https://user-images.githubusercontent.com/1795296/184433090-4e84cdbc-647c-492d-9177-1cb9f7af61bb.png)

**Comment by sretz** [comment]
What API number are you using in the emulator? And what API number is on your physical? Wondering if that makes a difference.

**Comment by xiz0r** [comment]
On the physical device I have API 30.
I just tested in the simulator with API 30 and it works fine.
In the simulator I tested with version 33 and it's wrong....
The image I uploaded is of the simulator with API 33.

```
<ItemGroup>
    <!-- Splash Screen -->
    <MauiSplashScreen Include="Resources\Splash\splash.svg" Color="#1F272F" />
```

**Comment by sretz** [comment]
it seems it always works with API 30, as that is my emulated and your physical and your working emulated. My physical is API 31. Perhaps API 30 is the last number it works correctly with.

**Comment by developer9969** [comment]
Hi @sretz 
I am having the same problems and this is my finding 

1. Android  introduced a new splash api.  
2. Xamarind Android team have done the binding - but no sample anywhere
3. Any app on android 12 will display the icon unless changes are made
4. What you have there will work for android <12 but not for android >=12  it will always show cut 
5. SplashActivity is no longer required
6. there are a couple of threads on the Xamarin android github issues have a look

There are new guidelines in place [https://developer.android.com/guide/topics/ui/splash-screen?authuser=1#:~:text=Splash%20Screen%20dimensions,-The%20splash%20screen&text=App%20icon%20with%20an%20icon,of%20192%20dp%20in%20diameter.](url)

@jfversluis  I know it might seem silly but in xamarin and maui all the apps in the wild have splash screens broken - displaying just the app's icon on android 12 it looks silly!!!  I know it's a xamarin android issue but we have not a single example or blog from you guys on how to implement a splash screen in xamarin android.

**Comment by sretz** [comment]
@developer9969 thanks for the sharing what you found, and that makes sense. Hopefully MS will take this seriously and fix it, as it might be a minor thing, but people who run the apps, this is up for a while and it's not a minor thing. It seems Android caused the problem, but MS needs to do something to work with it.

**Comment by mattleibow** [comment]
This will fix the dodgy images: https://github.com/dotnet/maui/pull/9509

**Comment by sretz** [comment]
@mattleibow great thanks, I'll give it a try when it gets pushed to VS, do you happen to know when that will be?

**Comment by mattleibow** [comment]
hopefully really, really soon. We are literally putting things together today.

**Comment by sretz** [comment]
@mattleibow thanks for the hard work

**Comment by mattleibow** [comment]
The issue with no images is duplicated from here https://github.com/dotnet/maui/issues/7404 and was fixed in this PR https://github.com/dotnet/maui/pull/8394

The issue with broken images is fixed here: #9509

The issue with the clipped round image is part of the new screenshot APIs Google has added. This is tracked here: #9794

**Comment by mattleibow** [comment]
And just and FYI, I found a nice and simple workaround for now until I can fix this in MAUI: https://github.com/dotnet/maui/issues/9794
